bin/lean-gdb.py

```diff
@@ -1,3 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+# Released under Apache 2.0 license as described in the file LICENSE.
+#
+# Author: Sebastian Ullrich
```

Comment:
> Huh, that's weird.

Oops, didn't notice the output goes to stdout, which is a different tab from gdb output in clion.  Works for me as well now.

> Yes, that works out of the box.

Ok, it's clear that I can cast it myself.  But how do you know it is a expr_const?